js/controller.js

```diff
@@ -656,6 +680,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.state.isPlayingAd = false;
       this.state.pluginsElement.removeClass("oo-showing");
       this.state.pluginsClickElement.removeClass("oo-showing");
+      this.showResponsiveDiv();
+      this.changeIMAiframeOpacity(0);
```

Comment:
I think it's because we manually change the opacity to 1 before that, it doesn't change back to 0 by itself.